README.md

```diff
@@ -1,44 +1,83 @@
 # GameOn Backend Api
-The Backend app for GameOn is hosted on Heroku, written in Python, using the Flask framework with data accessed through MongoDB.
-__GameOnBE: 'https://game-on-pro.herokuapp.com'__
+Host:       Heroku 
+Language:   Python
+Framework:  Flask 
+Database:   SQLAlchemy
+**GameOnBE: 'https://game-on-pro.herokuapp.com'**
 
 ### Formatting
-The GameOnBE App follows REST API convention, displaying data in JavaScript Object Notation(JSON).
+- The GameOnBE App follows REST API convention, displaying data in JavaScript Object Notation(JSON).
+- Each collection is returned within a 'data:' hash
```

Comment:
Can we change this to something like 'Each collection is returned within a hash that has a key of 'data' and a value of an array, containing objects returned in a hash format'